packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -54,17 +61,17 @@ export default function useFeedSettings(): FeedSettingsReturnType {
         { loggedIn: !!user?.id },
       );
 
-      if (user) {
-        return req;
-      }
-
```

Comment:
Realized we don't show the feed to anonymous users anymore so we don't need this condition here but rather to the `enabled` property.